pillar/top.sls

```diff
@@ -16,7 +16,9 @@ base:
     - datadog.elasticsearch-integration
   'roles:kibana':
     - match: grain
-    - kibana
+    - mitca
+    - elastic_stack.kibana
+    - nginx.kibana
```

Comment:
This should also have the `nginx` pillar assigned to pull in the logging configuration. So it would be:
```
- nginx
- nginx.kibana
```